src/main/kotlin/lotto/process/LottoScore.kt

```diff
@@ -0,0 +1,31 @@
+package lotto.process
+
+import lotto.model.LottoPrice
+import lotto.model.LottoPrize
+import lotto.model.LottoResult
+import lotto.model.LottoTicket
+
+class LottoScore {
+    fun compareNumber(winningTicket: LottoTicket, lottoTickets: List<LottoTicket>): List<LottoResult> {
+        val matchCounts = lottoTickets.map { lottoTicket ->
+            lottoTicket.get().mapNotNull { lottoNumber ->
+                winningTicket.get().find { it == lottoNumber }
+            }.size
```

Comment:
직접 비교하지 말고 `LottoTicket` 끼리 비교하도록 시켜보면 어떨까요?
```suggestion
                winningTicket.countIntersect(lottoNumber)
```